data/processed/gtr/total_gtr_projects_stem.csv

```diff
@@ -0,0 +1,501 @@
+year,nuts_id,nuts_year_spec,total_gtr_projects_stem
+2006,UKH2,2016,45
```

Comment:
Can we run autonuts here?

Reply:
The situation is the same as with the HEBCI data: we have reverse geocoded the data using the most recent shapefiles.

Reply:
OK I'll add this to #145